tests/sles4sap/qesapdeployment/test_cluster.pm

```diff
@@ -0,0 +1,33 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Test for qe-sap-deployment
+# Maintainer: QE-SAP <qe-sap@suse.de>, Michele Pagot <michele.pagot@suse.com>
+
+use strict;
+use warnings;
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use qesapdeployment;
+
+use constant GIT_CLONE_LOG => '/tmp/git_clone.log';
+
+sub run {
+    my ($self) = @_;
+    my $inventory = qesap_get_inventory(get_required_var('PUBLIC_CLOUD_PROVIDER'));
+    my $prov = get_required_var('PUBLIC_CLOUD_PROVIDER');
+
+    qesap_ansible_cmd(cmd => $_, provider => $prov, $_) for ('pwd', 'uname -a', 'cat /etc/os-release');
+    qesap_ansible_cmd(cmd => 'ls -lai /hana/', provider => $prov, filter => 'hana');
+    qesap_ansible_cmd(cmd => $_, provider => $prov, filter => 'vmhana01'), for ('crm status', 'crm_mon -R -r -n -N -1');
```

Comment:
I still think it would be nice to reuse `$hacluster::crm_mon_cmd` instead of `crm_mon -R -r -n -N -1` directly. That way if we need to change the command, we change it only on `lib/hacluster`

Reply:
I'd like to address it in a different commit